insights/client/config.py

```diff
@@ -435,6 +435,18 @@ def _boolify(v):
                                  for k, v in os.environ.items()
                                  if k.upper().startswith("INSIGHTS_") and
                                  k.upper() not in ignore)
+
+        for k in ['retries', 'cmd_timeout', 'http_timeout']:
+            if k in insights_env_opts:
+                v = insights_env_opts[k]
+                try:
+                    if k == 'http_timeout':
+                        insights_env_opts[k] = float(v)
+                    else:
+                        insights_env_opts[k] = int(v)
```

Comment:
What do you think of  `insights_env_opts[k] = float(v) if k == 'http_timeout' else int(v)`?

Reply:
much cleaner :+1: